python/ray/serve/_private/constants.py

```diff
@@ -78,6 +83,39 @@
     600000,
 ]
 
+
+def _parse_latency_buckets(bucket_str: str) -> list:
+    if not bucket_str.strip():
+        return DEFAULT_LATENCY_BUCKET_MS
```

Comment:
```suggestion
def _parse_latency_buckets(bucket_str: Optional[str]) -> List[float]:
    if bucket_str is None:
        return DEFAULT_LATENCY_BUCKET_MS
```
nit: `None` typically best practice for defaulting behavior (would also need to make the default for env var `None`)